scripts/generate_icons.sh

```
@@ -0,0 +1,43 @@
+    sizes[mipmap-mdpi]=48
+    sizes[mipmap-hdpi]=72
+    sizes[mipmap-xhdpi]=96
+    sizes[mipmap-xxhdpi]=144
+    sizes[mipmap-xxxhdpi]=192
+    
+    for dir in "${!sizes[@]}"; do
+        size=${sizes[$dir]}
+        output_path="android/app/src/main/res/$dir/ic_launcher.png"
+        
+        # Create a simple colored square with rounded corners
+        convert -size ${size}x${size} xc:"$BG_COLOR" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)